dart/dynamics/FreeJoint.cpp

```diff
@@ -575,6 +556,16 @@ const std::string& FreeJoint::getStaticType()
 }
 
 //==============================================================================
+void FreeJoint::setTransformFromChildBodyNode(const Eigen::Isometry3d& T)
+{
+  Joint::setTransformFromChildBodyNode(T);
+
+  // We update Jacobian here since the Jacobian of FreeJoint is a function of
+  // "T" but not of joint positions.
+  mJacobian = math::getAdTMatrix(T);
```

Comment:
:+1:  Then, I think we don't need this overriding function since `updateLocalJacobian()` is called in `Joint::setTransformFromChildBodyNode()`.